bundled/README.md

```diff
@@ -0,0 +1,6 @@
+# This folder contains third party projects used in AeroAcousticDG
+**Included projects have different copy rights compared to ExaDG. Their licences allow each project to be included here.**
```

Comment:
Q: Why do we bundle? Would git submodule the better approach so that we can update the package when needed? Have you modified the header file somehow?

Reply:
I didn't modify the header at all. Personally, I don't like git submodules too much, since the git commands get lengthy. Since the submodule will probably not need a lot of updates, this is probably not the case here?

Also I don't know how to deal with submodules etc, so I would need some help here.

Reply:
Let's keep it as it is!